src/Microsoft.Framework.Caching.SqlServer/SqlServerCacheExtensions.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.Caching.Distributed;
+using Microsoft.Framework.Caching.SqlServer;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    public static class SqlServerCacheExtensions
+    {
+        public static IServiceCollection AddSqlServerCache([NotNull] this IServiceCollection services)
+        {
+            services.AddOptions();
+            services.AddSingleton<IDistributedCache, SqlServerCache>();
+            return services;
+        }
+
+        public static void ConfigureSqlServerCache(
```

Comment:
OT: We seem to now have `AddX()` and `ConfigureX()` as separated things everywhere, probably with the only exception of `AddDbContext()` in EF? I just can't remember if/why we decided to have it this way. Right now it feels like collapsing the two methods into one if both exists would be much nicer. @lodejard @HaoK does anyone remember?